data/json/professions.json

```diff
@@ -696,7 +696,34 @@
   },
   {
     "type": "profession",
-    "id": "soldier",
+    "id": "major-general",
+    "name": "Major General",
+    "description": "You worked your way up through the ranks, from a no-name private, to a big shot Major General.  Now however, it is years since you last fired a weapon in anger, and you barely escaped your aides.  ",
```

Comment:
```suggestion
    "description": "You worked your way up through the ranks, from a no-name private, to a big shot Major General.  Now however, it is years since you last fired a weapon in anger, and you barely escaped your aides.",
```